Example/ViewController.swift

```diff
@@ -600,6 +600,13 @@ extension ViewController {
             
             var coordinates: [CLLocationCoordinate2D] = [location.coordinate]
             trackPolyline?.appendCoordinates(&coordinates, count: UInt(coordinates.count))
+            
+            if mapView?.userTrackingMode != .followWithCourse, location.horizontalAccuracy < 65.0, location.speed > 8.0, let camera = mapView?.camera {
+                mapView?.userTrackingMode = .followWithCourse
+                camera.altitude = 750.0
+                camera.pitch = 45.0
```

Comment:
Might make sense to align these values to CarPlay:

https://github.com/mapbox/mapbox-navigation-ios/blob/8a3aec8b152ad1cdede6ef8494300b3e2c4406f6/MapboxNavigation/CarPlayMapViewController.swift#L10